### ISSUE TITLE
Ad blocks download button

### ISSUE THREAD

**Issue by Marcisbee** [jakearchibald/svgomg#451]
No way to close that ad panel!

![image](https://github.com/user-attachments/assets/71b9b3b9-c86d-4e02-9d93-91aff679cffd)

**Comment by gabriel-rodrigues-42** [comment]
Exactly there is no way to save the image.

**Comment by FluroAsh** [comment]
Also seeing this issue, very annoying for an otherwise good tool! Had to modify the element with `display: none` just so I could download the SVG 🤣

**Comment by timoschwarzer** [comment]
Just fyi, if you're coming from svgomg dot net, that page seems to be a ripoff. Please use the page of this repo (https://jakearchibald.github.io/svgomg/) which does not have ads.

**Comment by methodofaction** [comment]
I'd suggest to use Google's feedback options to report as unusable so it drops from the ranking

<img width="618" alt="Image" src="https://github.com/user-attachments/assets/fc629d1e-3cfa-4f5b-bcee-a2d5f569a450" />

**Comment by jakearchibald** [comment]
Yeah, this is an open source project, so the person who's trying to make money from it isn't doing anything illegal. They're just being a bit morally shady.

To be clear, I have nothing to do with that site. The official URL for the site is https://jakearchibald.github.io/svgomg/, and there are no ads.
